Doc/whatsnew/3.10.rst

```diff
@@ -1000,6 +1000,10 @@ Optimizations
   performance in the common case of cache hits. This makes the interpreter 1.04 times faster
   in average (Contributed by Dino Viehland in :issue:`43452`)
 
+* A number of Python builtins instantiation times are now sped up by using :pep:`590` vectorcall protocol.
```

Comment:
I suggest to just that that the following built-in functions now support the faster vectorcall calling convention.